templates/https/agencies.html

```diff
@@ -1,8 +1,8 @@
 {% extends "layout.html" %}
```

Comment:
I adjusted the strings on the HTTPS screen to assume a little more knowledge by agencies, since we're focusing more on compliance and less on general HTTPS monitoring.